src/licensedcode/data/licenses/do-no-harm.yml

```diff
@@ -0,0 +1,8 @@
+key: do-no-harm
```

Comment:
Add a version e.g.  `key: do-no-harm-1.0`

Reply:
A proper version number hasn't been mentioned in this license, only "pre 1.0" is mentioned. They currently are working towards version 1.0. So should I just add `do-no-harm-pre1.0`?

Reply:
@JRavi2  the do-no-harm-license ([link](https://github.com/raisely/NoHarm/blob/publish/licenses/Apache-2.0-NoHarm.md)) is of version 1.0,so I think the name should be `do-no-harm-1.0`

Reply:
@Pratikrocks I believe you are referring to a different file. I've added [this](https://github.com/raisely/NoHarm/blob/publish/LICENSE.md) file, as in the readme of the repository, it has been mentioned to use this file.

Reply:
@JRavi2 .I think it the Readme is not updated, but the current version as per this [link](https://github.com/raisely/NoHarm/blob/publish/licenses/Apache-2.0-NoHarm.md) is 1.0

Reply:
The version mentioned there is 0.1. But, I guess it does look like a more updated version of the license. I'll change it here.